controllers/rcj_soccer_referee_supervisor/referee/consts.py

```diff
@@ -71,6 +71,8 @@ class LabelIDs(Enum):
     YELLOW_SCORE = 1
     TIME = 2
     EVENT_MESSAGES = 3
+    BLUE_TEAM = 10
+    YELLOW_TEAM = 11
```

Comment:
Why 10 and 11?

Reply:
Just to be on the safe side (since the `GOAL` was 4 in the other PR).

But we can easily refactor this.